File: SlagalicaSPA/src/app/games/models/question.ts

export class Question {
    constructor(
        public NumId: number,
        public Text: string,
        public Answers: string[],
        public CorrectAnswer : string,
    ) {
        this.Answers.push(CorrectAnswer);
        this.Answers.sort(
            () => Math.random() - 0.5
        )

    }

    public addCorrectAnswerToAnswers(){
        console.log('start')
        this.Answers.push(this.CorrectAnswer);
        this.Answers.sort(
            () => Math.random() - 0.5
        );
        console.log('end')

    }
}

File: Slagalica/WhoKnowsKnows/Entities/Question.cs
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson;

namespace WhoKnowsKnows.Entities
{
    public class Question
    {
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        public long NumId { get; set; }
        public string Text { get; set; }
        public List<string> Answers { get; set; }
        public string CorrectAnswer { get; set; }
    }
}
